### DISCUSSION TITLE
联通全部显示100%丢包

### DISCUSSION THREAD

**Discussion by mailuser963** [zdz/ServerStatus-Rust#208] [Q&A] [answered]
今天下午发现联通的丢包率全部是显示100%，联通时延测试全部显示1000ms。
电信和移动的丢包率和试验测试都显示正常。
而且联通线路确认可以正常连接各个节点。
是不是测试服务器出了问题？可以自己调整么？

**Comment by zdz** [comment, marked as answer]
见 https://github.com/zdz/ServerStatus-Rust/issues/206

**Reply by mailuser963** [reply]
搞定了，多谢。看来我这里的故障出现的时间晚两天，昨天看还是正常。
另外自定义ping测试站点有什么推荐的选择么？ 搜索了一下,结果都是些综合网站，找不到合适的域名或者ip。

**Reply by zdz** [reply]
都是自建维护，没有推荐哈

**Reply by mailuser963** [reply]
好的，我自己鼓捣。
没想到issue里已经有解答了，不耽误你的时间了，再次感谢。
